test.py

```
@@ -9094,3 +9094,1163 @@ async def test_cli_config_validate_fails_when_unconfigured(db_pool):
+        assert "id" in col_dict
+        assert "started_at" in col_dict
+        assert "ended_at" in col_dict
+        assert "summary" in col_dict
+        assert "summary_embedding" in col_dict
+        assert "time_range" in col_dict
+        
+        # Verify time_range is a tstzrange type
+        assert col_dict["time_range"]["data_type"] == "tstzrange"
+
+
+async def test_auto_episode_assignment_trigger(db_pool):
```

Ruff (0.14.10)

Redefinition of unused `test_auto_episode_assignment_trigger` from line 4489

(F811)